### PR TITLE
Add verified_only flag to get_followers()

### PR DESCRIPTION
**Pull request by pschroeder89**
## Description

I use this package to pull verified users for my business accounts' followers. This seems to be a common use case, so I figured I'd add these changes to the main repo.

## How Has This Been Tested?

The following command now returns only verified users for username:
```
followers_list = session.grab_followers(username=targetAccount, amount=amount, live_match=True, store_locally=False, verified_only=True)
```

Checked for 5 different accounts, all worked.

## Checklist:

- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have made corresponding changes to the documentation
- [X ] I have checked my code and corrected any misspellings
- [X ] I have performed a self-review of my own code
- [X ] My code follows the style guidelines of this project, `black -t py34`
- [X ] My changes generate no new warnings

### CODE REVIEW

File: instapy/relationship_tools.py [hunk 1]
```diff
@@ -185,6 +187,11 @@ def get_followers(
             page_info = data["user"]["edge_followed_by"]["page_info"]
             edges = data["user"]["edge_followed_by"]["edges"]
             for user in edges:
+                # If verified_only is True, determine if following user is verified and append to verified_followers
+                if verified_only:
+                    if user["node"]["is_verified"]:
```
Comment (elulcao) [inline instapy/relationship_tools.py:191]: nit: could be squashed this double `if` in a single one?

**Review by elulcao — APPROVED** [review]
Hello @pschroeder89 
Looks good to me, I added @breuerfelix as reviewer.
Just one thing, could you update `relationship-tools.md` to reflect this change,
Thanks 👍

File: instapy/relationship_tools.py (hunk 1 above)
Comment (pschroeder89) [reply instapy/relationship_tools.py:191]: done!

**Comment by breuerfelix** [comment]
yeah some notes in the readme would be awesome and also in the changelog :)  
also why can't we just use all_followers ? i think it would be better to use this list instead of the verified_list. cause if you pass in true, you just want verified ones.

@elulcao what about we merge this to a dev branch so we can add stuff like changelog and readme ourselfes before merging to master :)

**Comment by pschroeder89** [comment]
> yeah some notes in the readme would be awesome and also in the changelog :)
> also why can't we just use all_followers ? i think it would be better to use this list instead of the verified_list. cause if you pass in true, you just want verified ones.
> 
> @elulcao what about we merge this to a dev branch so we can add stuff like changelog and readme ourselfes before merging to master :)

Ok we now use all_followers to store just the verified users if verified_only is True. 
Also updated the readme and changelog.

File: instapy/relationship_tools.py (hunk 1 above)
Comment (pschroeder89) [reply instapy/relationship_tools.py:191]: Needed to break this up again since the logic is a bit different now

**Review by breuerfelix — APPROVED** [review]
awesome! i guess it is better with all_followers since there is a lot of logic afterwards aswell. now things won't break in some weird scenarios :) thanks for the PR !

**Comment by elulcao** [comment]
> yeah some notes in the readme would be awesome and also in the changelog :)
> also why can't we just use all_followers ? i think it would be better to use this list instead of the verified_list. cause if you pass in true, you just want verified ones.
> 
> @elulcao what about we merge this to a dev branch so we can add stuff like changelog and readme ourselfes before merging to master :)

Yeah, having a Dev Branch would be better for users. After a while we can push more fixes into the main branch. For the Dev branch, let me know if you have a chance to create it in the repo.

**Pull request merged**